sdk/resourcemanager/Azure.ResourceManager/src/ArmClient.cs

```diff
@@ -106,7 +106,7 @@ public ArmClient(TokenCredential credential, ArmClientOptions options)
             ClientOptions = options?.Clone() ?? new ArmClientOptions();
             Pipeline = ManagementPipelineBuilder.Build(Credential, options.Scope, options ?? ClientOptions);
 
-            _tenant = new TenantOperations(ClientOptions, Credential, BaseUri, Pipeline);
+            _tenant = new Tenant(ClientOptions, Credential, BaseUri, Pipeline);
             DefaultSubscription = string.IsNullOrWhiteSpace(defaultSubscriptionId)
                 ? GetDefaultSubscription()
                 : GetSubscriptions().GetIfExists(defaultSubscriptionId);
```

Comment:
GetIfExists's code seems to can return a null value but in reality, would have gotten 404 exception which is fine. Can we have a test case for an invalid subscription id?